src/main/scala/com/evolutiongaming/cluster/AdaptiveAllocationStrategy.scala

```diff
@@ -55,8 +55,11 @@ class AdaptiveAllocationStrategy(
   override def extractShardId(numberOfShards: Int): ShardRegion.ExtractShardId = {
     case x: ClusterMsg =>
       if (!x.isInstanceOf[PersistenceQuery]) {
-        increment(typeName, x.id)
-        metricRegistry.meter(s"persistence.$typeName.sender.${x.id}.$selfHost").mark()
+        if (!x.isInstanceOf[DoNotCount]) {
```

Comment:
let's consider case when we have `Command( Blackjack( PlaceBet | CheckIdle))` how can I disable counting for `CheckIdle` ?